public/assets/js/main.js

```diff
@@ -20,18 +22,53 @@
         }
     })();
 
-    (function selectInputContent() {
+    (function selectTextCopyToClipboard() {
         var selector = 'input[type="text"]';
         var el = document.querySelectorAll(selector);
+        var origHelpBlockText = 'Click to copy';
```

Comment:
Yeah, that is what I had, but it ESLint annoyed me with the no-loop-func errors so I hardcoded.

Agreed though it should be read with `innerHTML`.